frontend/src/stores/user.js

```diff
@@ -8,8 +9,8 @@ export const useUserStore = defineStore("user", {
    */
   state: () => ({
     users: [],
-    currentUser: null,
     dataLoaded: false,
+    currentUser: null, // Definimos currentUser en el estado
```

Comment:
Change comment to english

Reply:
Addressed in [commit](https://github.com/carlos18bp/gym_project/pull/5/commits/75f045424c57e6fbc9fb053661f5d23294a22f09)